be/src/olap/rowset/segment_v2/inverted_index_writer.cpp

```diff
@@ -695,6 +695,8 @@ Status InvertedIndexColumnWriter::create(const Field* field,
         M(FieldType::OLAP_FIELD_TYPE_DECIMAL128I)
         M(FieldType::OLAP_FIELD_TYPE_DECIMAL256)
         M(FieldType::OLAP_FIELD_TYPE_BOOL)
+        M(FieldType::OLAP_FIELD_TYPE_IPV4)
+        M(FieldType::OLAP_FIELD_TYPE_IPV6)
```

Comment:
Where is the code for transform encoding from little endian to big endian?

Reply:
in key_coder's full_encoding function.

Reply:
writer & reader part both has key_coder's full_encoding function before add_doc and seek in bkd

Reply:
ok